specs/UI/CountReportForDiagnosis.spec

```diff
@@ -0,0 +1,100 @@
+# Snomed Diagnosis Count report
+
+## Doctor should be able to check count reports for the malaria
+
+tags: snomed, ui
+
+* Login to Bahmni snomed location "General Ward" as a "receptionist"
+* Create new patient with random details and start an OPD visit
+* Logout and Login to Bahmni snomed location "General Ward" as a "doctor"
+* Open "Reports" module
+* Validate the count report for "SNOMED Diagnosis Count Report for Malaria" before adding the diagnosis "malaria" with snomed code "61462000"
+* Doctor add random snomed diagnosis for "malaria" with snomed code "61462000" using "name" search
+* Save data
+* Click back button
+* Click back button
+* Open "Reports" module
+* Select start date, end date and "HTML" format for "SNOMED Diagnosis Count Report for Malaria" and click on run button
+* Validate the count report generated for "malaria" if descendants of "malaria" is added.
```

Comment:
Dont use if statements. clearly write what is expected

Reply:
okay,will fix this

Reply:
fixed